pkg/apis/application/v1alpha1/chart_types.go

```diff
@@ -76,6 +76,8 @@ type ChartSpec struct {
 
 // +k8s:openapi-gen=true
 type ChartSpecConfig struct {
+	// Whether to install CRDs that might be supplied with the chart
+	SkipCRDInstall bool `json:"skipCrdInstall"`
```

Comment:
I think `SkipCRD` is enough WDYT?

```suggestion
	SkipCRD bool `json:"skipCrdInstall"`
```

Reply:
I am going to go with Ross suggestion and add an `Install` struct. Thanks 👍🏽

Reply:
Also see my comment https://github.com/giantswarm/apiextensions/pull/657/files#r558190783 which I think also applies here.